src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.fileReader;
+import core.basesyntax.dao.fileReaderImpl;
+import core.basesyntax.dao.writeReportImpl;
+import core.basesyntax.model.Account;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.service.action.*;
+import core.basesyntax.dao.writeReport;
```

Comment:
The class name `writeReport` does not follow Java naming conventions. Class names should be in PascalCase, such as `WriteReport`.